src/ebay-menu-button/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import initStoryshots from '@storybook/addon-storyshots'
+import { EbayMenuButton, EbayMenuButtonItem } from '..'
+
+initStoryshots({
+    config: ({ configure }) => configure(() => require('./index.stories.tsx'), module)
+})
+
+describe('<EbayMenuButton>', () => {
+    describe('on button click', () => {
+        let spy
+        beforeEach(() => {
+            spy = jest.fn()
+            spy.mockReset()
```

Comment:
Since you always re-create the spy on beforeEach, mockReset() is not necessary here.
Anyway, good tests 👍